tests/test_http_graphql_query.py

```diff
@@ -1190,8 +1190,12 @@ def test_graphql_functional_arguments_21(self):
     def test_graphql_functional_arguments_22(self):
         with self.assertRaisesRegex(
                 edgedb.QueryError,
-                r"invalid value for 'after'",
-                _line=5, _col=32):
+                # this error message is subpar, but this is what we get
+                # from postgres, because we transfer bigint values to postgres
+                # as strings
+                r'invalid input syntax for type bigint: "aaaaa"',
```

Comment:
There's a historical reason why this is a string and the short version of the story is that this was inspires by Relay Connections (see here: https://github.com/edgedb/edgedb/issues/187). Depending on whether we still find that style useful here, maybe we can change the type to be an integer like "first" and "last".

Reply:
Well, generally I don't think that pagination using `OFFSET` (which `after` and `before` are currently mapped to) is a good solution. Pagination should be coupled with sort order and use sorting key as `after` and `before`. But this is irrelevant to this PR. Please, create a new issue if you think this is worth changing now.